src/xb-database-manager.c

```diff
@@ -39,6 +39,7 @@
 #define STOPWORDS_METADATA_KEY "XbStopwords"
 #define QUERY_PARSER_FLAGS XAPIAN_QUERY_PARSER_FEATURE_DEFAULT | \
   XAPIAN_QUERY_PARSER_FEATURE_WILDCARD |\
+  XAPIAN_QUERY_PARSER_FEATURE_PURE_NOT |\
```

Comment:
I don't believe so no. We already strip out any xapian keywords (e.g. NOT, AND, OR) from the user's query. So we control when those operators get added and right now we don't use any pure NOT clauses. That being said, to be on the safe side, we probably shouldn't merge this to master until the `die-engine` feature branch gets merged to master in eos-knowledge-lib.